opencti-platform/opencti-front/src/private/components/common/files/workbench/WorkbenchFileContent.js

```diff
@@ -3686,7 +3686,7 @@ const WorkbenchFileContentComponent = ({
       {currentTab === 2 && renderRelationships()}
       {currentTab === 3 && renderContainers()}
       <Dialog
-        open={deleteObject}
+        open={!!deleteObject}
```

Comment:
Why is that necessary ? It seems that deleteobject is either null, undefined or an object ?

Reply:
`open` parameter should be a boolean, I had warnings in the console

Reply:
Oh, I see. In that case we might want to enforce a rule between this usage and the Boolean(deleteObject) one ? What do you like the most ?